docs/acceptance-criteria/uacs-rfc-19-random-beacon-v2.adoc

```diff
@@ -0,0 +1,503 @@
+  submitted. 
+
+| Group creation - DKG timeout
+| Anyone can send a transaction reporting DKG timeout.
+
+| Group creation - DKG timeout
+| Upon noticing the DKG timeout notification, the random beacon contract should
+  determine its validity.
+
+| Group creation - DKG timeout
+| When determining validity of the DKG timeout notification, the following
+  aspects should be taken into account: +
+  - does it reference the existing DKG request? (if not, then notification
+  invalid) +
+  - has the DKG timeout passed? (if not, then notification invalid) +
+  - is the sortition pool in a locked state (if not - meaning somebody already
+  unlocked it - then notification invalid).
+
+| Group creation - DKG timeout
+| If the DKG timeout notification was deemed justified, then: +
+  - the pool should be unlocked +
+  - the fixed amount reward (in T) should be sent from the maintenance pool to
+  the notifier.
+
+| Sortition pool - punishments
+| Member of the sortition pool can be punished with temporary exclusion from 
+  earning rewards as a result of: +
+  - misbehavior during the DKG submission +
+  - misbehavior during relay entry submission.
+
+| Sortition pool - punishments
+| Operators punished with removal from the group should not earn rewards for the
+  time of their absence in the pool.
+
+| Sortition pool - punishments
+| The operation removing members from the sortition pool should be as cheap as
+  possible so that the member submitting the relay entry or the DKG result does
+  not have to pay any additional costs for removing inactive members from the
+  pool.
+
+| Sortition pool - rewards
+| Operators in the sortition pool can earn T for multiple activities: +
+  - for staying in the pool (given weekly, proportionally to stake) +
+  - for successful submission of the DKG result (fixed value) +
+  - for successful unlocking of the sortition pool if DKG timed out.
+
+| Sortition pool - locking
+| When the sortition pool is locked, no operator can enter the pool.
+
+| Sortition pool - locking
+| When the sortition pool is locked, no operator can leave the pool.
+
+| Sortition pool - locking
+| When the sortition pool is locked, no operator can update its state.
+
+| Sortition pool - locking
+| Once the sortition pool gets unlocked, operators can leave the pool.
+
+| Sortition pool - locking
+| Once the sortition pool gets unlocked, new operators can join the pool.
+
+| Sortition pool - locking
+| Once the sortition pool gets unlocked, operators can update their state.
+
+| Sortition pool - locking
+| Operator's staked amount can change regardless if the pool is at the moment in
+  the locked or in the unlocked state (e.g. operator can be slashed during the
+  lock as a result of relay entry timeout).
+
+// TODO: Once starting values for governable parameters are agreed upon,
+// add them to the UACs
+
+| Governable parameters
+| The frequency of relay requests should be a governable parameter.
```

Comment:
Hmm.... no? Applications using the random beacon decide about it.

Reply:
Right, the frequency applies to creation of new groups. Improved the description in b0ed2dd.